specs/wp-comments-spec.js

```diff
@@ -71,4 +72,41 @@ describe( `[${ host }] Comments: (${ screenSize })`, function() {
 			);
 		} );
 	} );
+
+	describe( 'Commenting and replying to newly created post in Gutenberg Editor: @parallel @jetpack', function() {
+		step( 'Can login and create a new post', async function() {
+			this.loginFlow = new LoginFlow( driver, 'gutenbergSimpleSiteUser' );
+			await this.loginFlow.loginAndStartNewPost( null, true );
+			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+			await gEditorComponent.enterTitle( blogPostTitle );
+			return await gEditorComponent.enterText( blogPostQuote );
+		} );
+
+		step( 'Can publish and visit site', async function() {
+			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+			await gEditorComponent.publish( { visit: true } );
+		} );
+
+		step( 'Can post a comment', async function() {
+			const commentArea = await CommentsAreaComponent.Expect( driver );
+			return await commentArea._postComment( {
+				comment: dataHelper.randomPhrase(),
+				name: 'e2eTestName',
+				email: 'e2eTestName@test.com',
+			} );
+		} );
+
+		step( 'Can post a reply', async function() {
+			await driver.sleep( 10000 ); // Wait to not to post too quickly
```

Comment:
Looks like you are creating work for @juliaamosova 😃

Why this sleep is needed? Is it doable to wait for the comment to be posted?

Reply:
This is there, I believe, to keep spam triggers from being set off. If you post comments too quickly you'll get errors saying you need to slow down. This one has to be there.

Reply:
If the `sleep` statement is here to address the spam issue indeed, we'd need to keep it. I will, however, research if there is any way around that when get to work on this test.

Reply:
I just ran this a bunch of times without the sleep and it actually doesn't appear to be a spam thing, it is just waiting for the comment to show up. This should be updated to wait for the comment to show up

Reply:
OK I'll merge with the sleep since @juliaamosova is already going through all the sleeps

Reply:
Sounds good to me! I'll implement the wait and check for the comment as suggested by @bsessions85 which should address the sleep statement here.